approaches/byte-pair-encoding.md

```diff
@@ -5,3 +5,5 @@ layout: coming_soon
 title: Byte-pair encoding
 description: A data compression algorithm for machine translation
```

Comment:
Maybe it should be "for tokenization" or "for tokenization for machine translation"?  What do you think?